Common/Data/Market/QuoteBar.cs

```diff
@@ -272,7 +272,14 @@ public override void Update(decimal lastTrade, decimal bidPrice, decimal askPric
         /// <returns>Enumerable iterator for returning each line of the required data.</returns>
         public override BaseData Reader(SubscriptionDataConfig config, string line, DateTime date, bool isLiveMode)
         {
-            var csvLength = line.Split(',').Length;
+            var csvLength = 0;
+            for (var i = 0; i < line.Length && csvLength <= 5; i++)
```

Comment:
Won't this check always exit if `csvLength` is greater than 5?
I believe our QuoteBar standard format has 11 columns.